supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -5,5 +5,122 @@
+----
+
+Example doc text:
+----
+"Currently, you cannot use Orchestration (heat) templates with the director to deploy an overcloud that requires NFS as an Image service (glance) back end. There is currently no workaround for this issue."
+----
+
+----
+"The Compute services (nova) might fail to deploy because the nova_wait_for_compute_service script is unable to query the Nova API. If a remote container image registry is used outside of the undercloud, the Nova API service might not finish deploying in time. 
+Workaround: Rerun the deployment command, or use a local container image registry on the undercloud."
+----
+
+.Technology Preview
+
+Use present tense in the following format:
+----
+In <release>, a technology preview is available for <feature explanation>.
+----
+
+Example doc text: 
+----
+"In Red Hat OpenStack Platform 16.0, a technology preview is available for the Bare Metal Provisioning service (ironic) deployed on an IPv6 provisioning network for BMaaS (Bare Metal as-a-Service) tenants."
+----
+
+.Deprecated Functionality
+Warn users about the following deprecation stages:
+
+* Plan to deprecate
+* Deprecate
+* Plan to remove
+* Remove 
+
+If available, inform users of alternative capabilities and workarounds. Use the following format:
+
+Deprecation notice::
+----
+In <product> <release>, <name of capability> is deprecated. Bug fixes and support will be provided through the end of the <releasename or, if unknown, “a future”> lifecycle but no new feature enhancements will be made. (OPTIONAL: You can use <alternative capability> instead.) 
+----
+
+Removal notice::
+----
+In <product> <release>, <name of capability> is removed. Bug fixes and support will be provided through the end of the <releasename or, if unknown, “a future”> lifecycle but no new feature enhancements will be made. (OPTIONAL: You can use <alternative capability> instead.) 
+----
+
+Example doc texts:
+----
+"In Red Hat Openstack Platform (RHOSP) 14, the director graphical user interface will be deprecated. Bug fixes and support will be provided through the end of a future lifecycle but Red Hat will not implement new feature enhancements."
+----
+
+----
+"In Red Hat OpenStack Platform (RHOSP) 16, the Data Processing service (sahara) will be removed. Bug fixes and support will be provided through the end of the RHOSP 15  lifecycle but Red Hat will not implement new feature enhancements."
+----
+
+.Survival tips
+
+- Remove any asterisks (`*`) when you edit the doc text, because they might interfere with the draft text formatting.
+- You can add links in doc text, for example if you need to link to more information in the doc set or to a KBase article.
+- When you review Doc Text and save the BZ, all the people in the CC list receive an email notification, which can act as an indirect request for tech review on the edited text. However, if you are unsure of the text, contact the reporter or the person who wrote the draft directly and ask for a tech review.
```

Comment:
Delete "However" because you normally do not start a sentence with a conjunction.

Reply:
Thanks Les, this little section is actually removed. It was something someone else wrote as part of the Release Monitor Playbook that I copied across accidentally at first. So it should be gone now if you look in the **Files changed** tab.
Thanks so much for your review! I implemented all your other changes.